### DISCUSSION TITLE
U8G2_ST75256_JLX256160_F_HW_I2C and the reset pin

### DISCUSSION THREAD

**Discussion by dynek** [olikraus/u8g2#1824] [Q&A] [answered]
Hello,

The declaration of U8G2_ST75256_JLX256160_F_HW_I2C has reset pin set to U8X8_PIN_NONE.
It is not clear to me if I need to set a reset pin or if it's not needed by the display?

How can I determine?

Thank you

**Comment by olikraus** [comment, marked as answer]
If in doubt, better use the reset pin. 
Basically everything should be mentioned in the datasheet if your display.
